plaso/parsers/winevtx.py

```diff
@@ -277,6 +277,10 @@ def _ParseRecords(self, parser_mediator, evtx_file):
         parser_mediator.ProduceExtractionWarning(
             'unable to parse event record: {0:d} with error: {1!s}'.format(
                 record_index, exception))
+      except ElementTree.ParseError as exception:
```

Comment:
Instead of adding another except, please change the except at line 276 to:
except (IOError, ElementTree.ParseError) as exception: